SerializerManager.js

```diff
@@ -14,15 +14,21 @@ class Serializer {
 class SerializerManager {
   constructor (configuration) {
     this.serializers = configuration.serializers
+    this.cache = configuration.cache
   }
 
   findSerializer (types) {
+    const cacheValue = this.cache[types]
+
+    if (cacheValue) return cacheValue
```

Comment:
I agree that the `-100` coverage check the cache statement, I would just add a test that checks that the cache is filled and don't pollute (ex: 2 call different routes with same `Accept` expects only 1 entry in the cache).

Adding a test with an handler that check the `reply.serializer.cache` object (that should be accessible by the user) would add a really valuable check to avoid bad side cases when implementing the per-route-serialization

Reply:
done